src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -559,3 +559,89 @@ def _patch(self, crd, crd_name, cr_name, func):
                 "Failed to update {}/{}: {}".format(crd_name, cr_name, e))
 
         return "Success"
+
+    def get_ceph_image(self):
+        image = "ceph/ceph:v14.2.6"
+
+        try:
+            api_response = self.coreV1_api.list_namespaced_pod(self.rook_env.namespace, label_selector="app=rook-ceph-mon", timeout_seconds=10)
+            if api_response.items:
+                image = api_response.items[-1].spec.containers[0].image
+        except ApiException as e:
+            log.warning("Error getting ceph image: {}, using <{}> as default".format(e, image))
+
+        return image
+
+    def blink_light(self, ident_fault, on, locs):
+        # type: (str, bool, List[DeviceLightLoc]) -> string
+        operation_id = str(hash(locs))
```

Comment:
should we add `ident_fault` and `on` to the hash?

Reply:
I think this will keep safe the last "light" operation executed over a physical disk.  In the case of several operations launched simultaneously or almost simultaneously, probably the right thing to do is to deem as valid only the last one. (independently if this operation is to switch on or off the light). 
And as we use only the identification light  neither is needed to add the type of operation to the hash